tests/sqlalchemy_dialect_compliance/test_dialect_compliance.py

```diff
@@ -42,8 +46,434 @@
     TimestampMicrosecondsTest as _TimestampMicrosecondsTest,
 )
 
+from sqlalchemy.testing.suite.test_types import (
+    ArrayTest,
+)
+
+from sqlalchemy.testing.suite.test_reflection import (
+    BizarroCharacterFKResolutionTest,
+    ComponentReflectionTest,
+    OneConnectionTablesTest,
+    HasTableTest,
+)
+
+if packaging.version.parse(sqlalchemy.__version__) >= packaging.version.parse("2.0"):
+    import uuid
+    from sqlalchemy.sql import type_coerce
+    from sqlalchemy import Uuid
+    from sqlalchemy.testing.suite import (
+        TrueDivTest as _TrueDivTest,
+        IntegerTest as _IntegerTest,
+        NumericTest as _NumericTest,
+        DifficultParametersTest as _DifficultParametersTest,
+        FetchLimitOffsetTest as _FetchLimitOffsetTest,
+        PostCompileParamsTest,
+        StringTest as _StringTest,
+        UuidTest as _UuidTest,
+    )
+
+    class TimestampMicrosecondsTest(_TimestampMicrosecondsTest):
+        data = datetime.datetime(2012, 10, 15, 12, 57, 18, 396, tzinfo=pytz.UTC)
+
+        def test_select_direct(self, connection):
+            # This func added because this test was failing when passed the
+            # UTC timezone.
+
+            def literal(value, type_=None):
+                assert value == self.data
+
+                if type_ is not None:
+                    assert type_ is self.datatype
+
+                import sqlalchemy.sql.sqltypes
```

Comment:
not sure this import is necessary.
we import this at the top level before we do any checks for sqlalchemy versions, etc.